client/src/components/Slider/Slider.css

```diff
@@ -0,0 +1,38 @@
+.slider__cards {
+        display: flex;
+}
+
+.PersonCard{
+        width: 100%;
+        margin: 0 2%;
+}
```

Comment:
Don't overwrite the styling of another component. That will become almost impossible to debug later. Can you make sure you're not doing this elsewhere in the PR?